Library/Homebrew/formula.rb

```diff
@@ -1017,8 +1017,7 @@ def service_name
   # The generated launchd {.plist} file path.
   sig { returns(Pathname) }
   def plist_path
-    # TODO: Add deprecation
-    # odeprecated "formula.plist_path", "formula.launchd_service_path"
+    odeprecated "formula.plist_path", "formula.launchd_service_path"
```

Comment:
Feels like this could have all references just changed over easily enough.

Reply:
I've seen some usages in Homebrew/core when converting services so I'd recommend keeping the deprecation.